datastacks/config_class.py

```diff
@@ -0,0 +1,48 @@
+from datetime import date
+from typing import List, Optional
+
+from pydantic import BaseModel, Field
+
+
+class IngestConfig(BaseModel):
+    dataset_name: str = Field(
+        description="Dataset name, used to derive pipeline and linked service names."
+    )
+    pipeline_description: str = Field(
+        description="Description of the pipeline to be created."
+    )
+    data_source_type: str = Field(description="Datasource type eg. azure_sql.")
```

Comment:
We could check it against the supported values for data_source_type and also present in readme/help what source types are valid.

Reply:
Do you have a link to the list of supported values?

Reply:
I don't have and exact working example. I was thinking of either using Enum or it looks like there are also validators available in Pydantic (haven't used them before though): https://docs.pydantic.dev/latest/usage/validators/

With Enum I imagined something of this sort:

```
class DataSource(Enum):
    AZURE_SQL = 'azure_sql'
    MS_SQL = 'ms_sql'

class Config(BaseModel):
    color: DataSource
```

Reply:
Sorry if I wasn't clear, I meant the supported values not how to validate. I know how to validate them I just can't find what's supported.

Reply:
Oh, sure :) AFAIK `azure_sql` is the only supported data type as of now. I guess we'd need to add more if we add them, we haven't tested for anything else yet and there might be some updates required to support more.

Reply:
perfect. I thought that but wanted to double check, thanks!

Reply:
Done